CefSharp/DependencyChecker.cs

```diff
@@ -69,22 +69,36 @@ public static class DependencyChecker
         {
             "CefSharp.Core.dll",
             "CefSharp.dll",
-            "CefSharp.BrowserSubprocess.Core.dll",
```

Comment:
Ok I misunderstood.  I thought that was a DLL for CefSharp, not the BrowserSubprocess.